nitter/data/nitter/nitter.conf

```diff
@@ -0,0 +1,45 @@
+[Server]
```

Comment:
I don't think this file is needed either as this also already bundled into the image (https://github.com/goodtiding5/docker-nitter/blob/01a2e49c7a050edbe4b06bd49bb97b12fcac2a17/Dockerfile#L43). I can see that you've filled out the values already, but I think we can leverage the env. vars. that the `entrypoint.sh` script is looking to do the templating for us?